lib/src/main/java/org/esbtools/eventhandler/PollingDocumentEventProcessorRoute.java

```diff
@@ -51,12 +52,24 @@ public PollingDocumentEventProcessorRoute(DocumentEventRepository documentEventR
         this.batchSize = batchSize;
         this.documentEndpoint = documentEndpoint;
         this.failureEndpoint = failureEndpoint;
+        this.routeId = "documentEventProcessor-" + id;
     }
 
+    public PollingDocumentEventProcessorRoute(DocumentEventRepository documentEventRepository,
+    		Duration pollingInterval, int batchSize, String documentEndpoint,
+            String failureEndpoint, String routeId) {
+        this.documentEventRepository = documentEventRepository;
+        this.pollingInterval = pollingInterval;
+        this.batchSize = batchSize;
+        this.documentEndpoint = documentEndpoint;
+        this.failureEndpoint = failureEndpoint;
+        this.routeId = routeId;
+    }
+    
     @Override
     public void configure() throws Exception {
         from("timer:pollForDocumentEvents" + id + "?period=" + pollingInterval.toMillis())
```

Comment:
I think let's use the routeId here so it's a little nicer to find logs for the timer for a given route, as in `"timer:pollForDocumentEvents-" + routeId + ...`